src/main/java/com/devonfw/ide/sonarqube/common/impl/SonarDevon4jPlugin.java

```diff
@@ -1,30 +1,94 @@
 package com.devonfw.ide.sonarqube.common.impl;
-
 import org.sonar.api.Plugin;
 import org.sonar.api.PropertyType;
 import org.sonar.api.config.PropertyDefinition;
-
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 /**
  * The {@link Plugin} to integrate devonfw architecture rules into SonarQube.
  */
 public class SonarDevon4jPlugin implements Plugin {
-
   static final String CONFIG_KEY = "sonar.devon.config";
-
   static final String FORBIDDEN_CONF_KEY = "sonar.devon.forbiddenConf";
-
   static final String DISABLED = "Disabled";
-
   static final String ISSUES_SEVERITY_KEY = "sonar.Devon.preview.issuesSeverity";
-
+  private static final String QUALINSIGHT = "qualinsight-plugins-sonarqube-smell-plugin";
+  private static final String PMD = "sonar-pmd-plugin";
+  private static final String CHECKSTYLE = "checkstyle-sonar-plugin";
+  private static final String FINDBUGS = "sonar-findbugs-plugin";
+  static List<String> FORBIDDEN_REPO_KEYS = new ArrayList<>();
+  private File pluginDirectory;
+  private List<String> pluginList;
+  private List<String> requiredPlugins = Arrays.asList("qualinsight-plugins-sonarqube-smell-plugin",
```

Comment:
Putting the required plugins inside a list is fine, but then we wouldn't need them double. As it is now, we have individual strings for all the plugins we need, plus this list. One of them should be enough